npm/qsharp/ux/circuit-vis/contextMenu.ts

```diff
@@ -47,32 +47,36 @@ const addContextMenuToHostElem = (
     });
 
     const adjointOption = _createContextMenuItem("Toggle Adjoint", () => {
-      selectedOperation.isAdjoint = !selectedOperation.isAdjoint;
+      if (selectedComponent.type !== "Operation") return;
+      selectedComponent.isAdjoint = !selectedComponent.isAdjoint;
```

Comment:
If you're going for minimal changes, I feel like it's less wordy to write:

```suggestion
        selectedOperation.isAdjoint = (selectedComponent.type !== "Operation" || !selectedOperation.isAdjoint);
```

If you're going for _overall_ conciseness, this function probably can be refactored a bit to be shorter so you don't have `addControlOption` etc even defined for a measurement.

Reply:
I've tried reordering the function a bit. Let me know if this makes things clearer for this function.